src/imperator/data/deity.rs

```diff
@@ -31,8 +31,13 @@ impl DbKind for Deity {
         let mut sc = ScopeContext::new(Scopes::Country, key);
 
         data.verify_exists(Item::Localization, key);
-        let loca = format!("omen_{key}_desc");
+        let key_str = key.to_string();
+        // Slice "deity_" off the front of the key
+        let diety_key = key_str.char_indices().nth(6).map_or(key_str.len(), |(i, _)| i);
```

Comment:
This is a job for strip_prefix :)

You can do it like
```
if let Some(no_deity_key) = key.as_str().strip_prefix("deity_") {
    ... check loca
} else {
    ... something -- maybe warn that key should start with deity_ ?
}
```

However it might be good to check if that's really what's happening here. I've never before seen Pdx manipulate key names like that. Could there be a separate omen_ key in a different folder, that just happens to have the same names as the deities? I've had a couple of cases like that in ck3.

Also check the spelling, it's deity not diety.